internal/config/config.go

```diff
@@ -1043,6 +1050,14 @@ type AzureSEVSNP struct {
 	AMDRootKey Certificate `json:"amdRootKey" yaml:"amdRootKey"`
 }
 
+// setWantLastestFalse sets the WantLatest field to false for all versions in order to unmarshal the numerical versions instead of the string "latest".
+func (c *AzureSEVSNP) setWantLastestFalse() {
```

Comment:
```suggestion
// setWantLatestFalse sets the WantLatest field to false for all versions in order to unmarshal the numerical versions instead of the string "latest".
func (c *AzureSEVSNP) setWantLatestFalse() {
```